text/0000-consortium.md

```diff
@@ -0,0 +1,69 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Relevant Project: (e.g. EaselJS, SoundJS, PreloadJS, TweenJS, ...)
+
+# CreateJS Consortium
```

Comment:
Is consortium even the right name?

We're trying to prove to @gskinner that the CreateJS projects can survive becoming community-led, as opposed to being led by themselves.

Reply:
Here is Webster's definition:

Consortium: an agreement, combination, or group (as of companies) formed to undertake an enterprise beyond the resources of any one member.  E.g. A consortium of researchers decoded the honeybee genome.

Sounds friendly in that example... but usually it sounds business-like and might get confused with a Cartel, etc.  Anyway - it has alliteration CreateJS Consortium.  Its definition fits.  It sounds official and it holds weight.  But... is it cool?  Is it nerdy enough?  It is not as fun as CreateJS Crew... etc.  Maybe all the consortium does is put together the plan and that includes a name for the future!

Reply:
Not that I'm that bothered by it, just giving my 2c: I've not heard 'Consortium' used on open source projects. Off the top of my head, there's usually a community, and some kind of core leadership. In my opinion is that there's great benefit in helping the community getting involved. This means removing barriers to contribution from a range of expertise levels. This might mean defining what can be done, how to do it, and how it will be lead/managed/guided/moderated etc by the core team.

Is the core team the consortium?

Anyway - again, I'm not too fussed by naming.